.github/workflows/release.yml

```diff
@@ -7,27 +7,119 @@ on:
         description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
         default: '7.y.z'
+
+env:
+  IMAGE: quay.io/eclipse/che-dashboard
+
 jobs:
-  build:
-    name: Create Release
-    runs-on: ubuntu-latest
+
+  build-images:
+    runs-on: ubuntu-18.04
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [amd64,arm64,ppc64le,s390x]
+    outputs:
+      amd64: ${{ steps.result.outputs.amd64 }}
+      arm64: ${{ steps.result.outputs.arm64 }}
+      ppc64le: ${{ steps.result.outputs.ppc64le }}
+      s390x: ${{ steps.result.outputs.s390x }}
     steps:
-      - uses: actions/checkout@v2
-      - name: Login to docker.io
+      -
+        name: "Checkout Che Dashboard source code"
+        uses: actions/checkout@v2
+      -
+        name: "Set up QEMU"
+        uses: docker/setup-qemu-action@v1
+      -
+        name: "Set up Docker Buildx ${{ matrix.arch }}"
+        uses: docker/setup-buildx-action@v1
+      -
+        name: "Docker quay.io Login"
         uses: docker/login-action@v1
         with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      -
+        name: "Docker docker.io Login"
+        uses: docker/login-action@v1
+        with:
+          registry: docker.io
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-          registry: docker.io
-      - name: Login to quay.io
+      -
+        name: "Build and push ${{ matrix.arch }}"
+        uses: docker/build-push-action@v2
+        with:
+          context: .
```

Comment:
Are you missing a cache step here?

https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/ci-multiarch.yaml#L65-L66

Reply:
@nickboldt Actually, I have not observed caching for release workflow in other packages so had not included it here. But if we need to add caching then will it use the same image as used in [ci-multiarch.yaml](https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/ci-multiarch.yaml) i.e docker.io/cheincubator/che-dashboard:cache ?

Reply:
yeah, I suppose so... unless that will cause collisions with the nightly builds? @sleshchenko  your thoughts?

Reply:
Well, I'm not sure but I think caching in the CI-multiarch is broken since all archs which are run in parallel use the same tag for cache
```
> docker pull  docker.io/cheincubator/che-dashboard:cache
cache: Pulling from cheincubator/che-dashboard
no matching manifest for linux/amd64 in the manifest list entries
```

As about caching for release, I assume it's redundant, the only stuff can be cached - dependencies, which depends on 
```
COPY package.json /dashboard/
COPY yarn.lock /dashboard/
```
But since package.json has [specified version](https://github.com/eclipse/che-dashboard/blob/master/package.json#L3), which is different from release to release, and from next builds, the cache won't be used.

The answer for cache here is: 
- how many times we trigger release build for the same version?
I think it must be 1)